2020-07-21 09:46:46,908 - teradata.udaexec - INFO - Initializing UdaExec...
2020-07-21 09:46:46,910 - teradata.udaexec - INFO - Reading config files: ['C:\\etc\\udaexec.ini: Not Found', 'C:\\Users\\vkadimis\\udaexec.ini: Not Found', 'C:\\Users\\vkadimis\\Desktop\\Vamsi Python Scripts\\Export Insert Statements\\udaexec.ini: Found']
2020-07-21 09:46:46,910 - teradata.udaexec - INFO - Found run number file: "C:\Users\vkadimis\Desktop\Vamsi Python Scripts\Export Insert Statements\.runNumber"
2020-07-21 09:46:46,910 - teradata.udaexec - INFO - Cleaning up log files older than 90 days.
2020-07-21 09:46:46,911 - teradata.udaexec - INFO - Removed 0 log files.
2020-07-21 09:46:48,199 - teradata.udaexec - INFO - Checkpoint file not found: C:\Users\vkadimis\Desktop\Vamsi Python Scripts\Export Insert Statements\impactAnalysis.checkpoint
2020-07-21 09:46:48,199 - teradata.udaexec - INFO - No previous checkpoint found, executing from beginning...
2020-07-21 09:46:48,199 - teradata.udaexec - INFO - Execution Details:
/********************************************************************************
 * Application Name: impactAnalysis
 *          Version: 2
 *       Run Number: 20200721094646-17
 *             Host: VKADIMIS-4NA7P
 *         Platform: Windows-10-10.0.17134-SP0
 *          OS User: vkadimis
 *   Python Version: 3.7.6
 *  Python Compiler: MSC v.1916 64 bit (AMD64)
 *     Python Build: ('tags/v3.7.6:43364a7ae0', 'Dec 19 2019 00:42:30')
 *  UdaExec Version: 15.10.0.21
 *     Program Name: C:\Users\vkadimis\Desktop\Vamsi Python Scripts\Export Insert Statements\export_insert_statements.py
 *      Working Dir: C:\Users\vkadimis\Desktop\Vamsi Python Scripts\Export Insert Statements
 *          Log Dir: C:\Users\vkadimis\Desktop\Vamsi Python Scripts\Export Insert Statements\logs
 *         Log File: C:\Users\vkadimis\Desktop\Vamsi Python Scripts\Export Insert Statements\logs\impactAnalysis.20200721094646-17.log
 *     Config Files: ['C:\\etc\\udaexec.ini: Not Found', 'C:\\Users\\vkadimis\\udaexec.ini: Not Found', 'C:\\Users\\vkadimis\\Desktop\\Vamsi Python Scripts\\Export Insert Statements\\udaexec.ini: Found']
 *      Query Bands: ApplicationName=impactAnalysis;Version=2;JobID=20200721094646-17;ClientUser=vkadimis;Production=False;udaAppLogFile=C:\Users\vkadimis\Desktop\Vamsi Python Scripts\Export Insert Statements\logs\impactAnalysis.20200721094646-17.log;UtilityName=PyTd;UtilityVersion=15.10.0.21
********************************************************************************/
2020-07-21 09:46:48,201 - teradata.udaexec - INFO - Creating connection: {'method': 'odbc', 'system': 'TDPROD', 'username': 'vkadimis', 'password': 'XXXXXX', 'driver': 'Teradata', 'authentication': 'LDAP'}
2020-07-21 09:46:48,350 - teradata.tdodbc - INFO - Available drivers: SQL Server, Oracle in OraClient12Home1, BMF SQLite3 ODBC Driver, DataDirect 7.1 DB2 Wire Protocol, DataDirect 7.1 Informix Wire Protocol, DataDirect 7.1 Sybase Wire Protocol, DataDirect 7.1 SQL Server Wire Protocol, DataDirect 7.1 dBaseFile, DataDirect 7.1 FoxPro 3.0, DataDirect 7.1 MySQL Wire Protocol, DataDirect 7.1 New SQL Server Wire Protocol, DataDirect 7.1 Greenplum Wire Protocol, Informatica MongoDB ODBC Driver, DataDirect 7.1 Oracle Wire Protocol, Informatica Data Services ODBC Driver 10.1.0, Informatica Cassandra ODBC Driver, Teradata, Teradata 7.1 DB2 Wire Protocol, Teradata 7.1 Oracle, Teradata 7.1 Oracle Wire Protocol, Teradata 7.1 SQL Server Legacy Wire Protocol, Teradata 7.1 SQL Server Wire Protocol, DataDirect 8.0 New SQL Server Wire Protocol, DataDirect 7.1 PostgreSQL Wire Protocol, DataDirect 8.0 Oracle Wire Protocol, Informatica Data Services ODBC Driver 10.4.0
2020-07-21 09:46:54,465 - teradata.udaexec - INFO - Connection successful. Duration: 6.265 seconds. Details: {'method': 'odbc', 'system': 'TDPROD', 'username': 'vkadimis', 'password': 'XXXXXX', 'driver': 'Teradata', 'authentication': 'LDAP'}
2020-07-21 09:46:54,775 - teradata.udaexec - ERROR - Query Failed! Duration: 0.297 seconds, Query: sel * from etlvwdb_ts3.dw_job_streams where job_stream_name in ('wf_FDRR_PARTIAL_REFRESH_ACTIVATE','wf_FDRR_PARTIAL_REFRESH_DEACTIVATE');
Traceback (most recent call last):
  File "C:\Users\vkadimis\AppData\Local\Programs\Python\Python37\lib\site-packages\teradata\udaexec.py", line 790, in _execute
    func(query, params, **kwargs)
  File "C:\Users\vkadimis\AppData\Local\Programs\Python\Python37\lib\site-packages\teradata\tdodbc.py", line 614, in execute
    checkStatus(rc, hStmt=self.hStmt, method="SQLExecDirectW")
  File "C:\Users\vkadimis\AppData\Local\Programs\Python\Python37\lib\site-packages\teradata\tdodbc.py", line 231, in checkStatus
    raise DatabaseError(i[2], u"[{}] {}".format(i[0], msg), i[0])
teradata.api.DatabaseError: (3802, "[42S02] [Teradata][ODBC Teradata Driver][Teradata Database] Database 'etlvwdb_ts3' does not exist. ")
